tests/test_sbr_calculator.py

```diff
@@ -0,0 +1,129 @@
+from src.smart_building_rating_calculator.inputs import (
+    EVChargerPower,
+    BatterySize,
+    SolarInverterSize,
+    HeatingSource,
+    HotWaterSource,
+)
+from src.smart_building_rating_calculator.flex_archetype import FlexArchetypes
+from src.smart_building_rating_calculator.main import sbr_score
+
+
+def test_high_sbr_gold_flexer():
```

Comment:
All these tests are testing the same function. I'd wrap them up in 1 test class:

class Test_sbr_score:
    def test_gold_flexer():
    def test_strong_flexer():

etc

Reply:
You should also run tests for the intermediate functions in between - and not rely on just a single test on the final function.

Reply:
Added more extensive testing functions & units tests.